modules/core/include/opencv2/core/matx.hpp

```diff
@@ -1275,6 +1275,48 @@ Matx<_Tp, m, n> operator * (double alpha, const Matx<_Tp, m, n>& a)
     return Matx<_Tp, m, n>(a, alpha, Matx_ScaleOp());
 }
 
+template<typename _Tp, int m, int n> static inline
+Matx<_Tp, m, n>& operator /= (Matx<_Tp, m, n>& a, int alpha)
```

Comment:
Matx usually makes sense (used) with floating point numbers (float/double).
So "int" scalar looks out of scope here. Do we really need it?

For floating point computations we can replace multiple divisions by multiplications (they are still more effective than divisions):
```
_Tp alpha_inv = (_Tp)1.0f / alpha;
for( int i = 0; i < m*n; i++ )
        a.val[i] = a.val[i] * alpha_inv;
```

`saturate_cast()` is not used for floating point numbers too.